src/main/java/uk/gov/hmcts/darts/task/runner/impl/CaseExpiryDeletionAutomatedTask.java

```diff
@@ -30,6 +31,7 @@ public class CaseExpiryDeletionAutomatedTask
 
     private final CurrentTimeHelper currentTimeHelper;
     private final DataAnonymisationService dataAnonymisationService;
+    private final HearingsService hearingsService;
```

Comment:
It's preferable to avoid directly injecting services from other modules.  Suggest creating a `HearingApi` interface (and impl) in `src/main/java/uk/gov/hmcts/darts/hearings/api`, and a `removeMediaLinkToHearing(int courtCaseId)` method that simply delegates to the needed functionality in `HearingService`.

You can then inject `HearingApi` here , and call `hearingApi.removeMediaLinkToHearing(courtCaseId)` on L70.